### PR TITLE
docs: First cut at 0.3.2 CHANGELOG features

### PR DESCRIPTION
**Pull request by richb-hanover**
Juicy stuff available. Let's tell about it (for the especially intrepid...)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,59 @@
...
 
 0.3.1 brings a couple of small improvements and fixes.
 
 **Features**:
 
 - Support for using s-strings for `from` (#1197, @aljazerzen)
+
   ```
-  from s"SELECT * FROM employees WHERE foo > 5"
+  from s"SELECT \* FROM employees WHERE foo > 5"
```
Comment (max-sixty) [inline CHANGELOG.md]: What was wrong with this? It seems to show OK at 
image

https://github.com/prql/prql/blob/main/CHANGELOG.md

...and with the change, it shows the `\`

File: CHANGELOG.md [hunk 2]
```diff
@@ -47,14 +91,14 @@ We've had to make some modest breaking changes for 0.3:
...
+```diff
+from employees
+-join positions [id]
++join positions [==id]
+```
 
-  The existing approach is ambiguous to the compiler — `id` could be a boolean
-  column.
+The existing approach is ambiguous to the compiler — `id` could be a boolean
+column.
```
Comment (max-sixty) [inline CHANGELOG.md]: These should be indented, since they're part of the bullet above

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,15 +1,59 @@
...
 
+## 0.3.2 — [unreleased]
+
+**Features**:
+
+- S-strings can [produce a full table.](https://prql-lang.org/book/language-features/s-strings.html#prql-3)
+- _Experimental:_ `switch` statement sets a variable to a value based on one of several expressions.
+  No page in the docs yet, but
+  [see this discussion](https://github.com/prql/prql/issues/1286#issue-1501645497)
+  for usage and the current syntax. _Note: this syntax may change._
```
Comment (max-sixty) [inline CHANGELOG.md:11]: Very nice! Thanks a lot for adding these!

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,15 +1,59 @@
...
+- _Experimental:_ `union` statement.
+  No page in the docs yet, but [see this PR](https://github.com/prql/prql/pull/894#issuecomment-1353548853) for usage.
+
+  ```
+  from employees
+  concat managers
+  union other_employees
+  ```
+
+- Any other `feat:` commits?
```
Comment (max-sixty) [inline CHANGELOG.md:31]: Not on you to add them — do what you're happy doing — but some other ones we can add:
- Playground now live-queries some sample data https://github.com/prql/prql/commit/fadd91762705f791448a95e4be3798e90fb05234
- I add commits from newer contributors: https://github.com/prql/prql/commit/8f57f437dd05b161c9ff14e96b01f0f0addd620e
- Good docs improvement! https://github.com/prql/prql/commit/a90d5502b092e3369a38816de36b8782c9ea823a

**Review by max-sixty — APPROVED** [review]
This is helpful, thanks @richb-hanover

**Comment by snth** [comment]
Thanks @richb-hanover .

I've created a [0.4 Milestone](https://github.com/prql/prql/milestone/4) for the upcoming release and tagged some issues that are already included or might be included. That might be a useful list to review for the CHANGELOG.

File: CHANGELOG.md (hunk 2 above)
Comment (richb-hanover) [reply CHANGELOG.md]: My version of Prettier (VScode) seems to monkey with this. (More research needed) I fixed it by hand and re-worded to eliminate possible confusion

File: CHANGELOG.md (hunk 4 above)
Comment (richb-hanover) [reply CHANGELOG.md:31]: OK. Thanks!

**Comment by max-sixty** [comment]
Super, thank you @richb-hanover !

**Pull request merged**